### PR TITLE
add fluxette example

### PR DESCRIPTION
**Pull request by alkhe**
Adding Fluxette to the list.

### CODE REVIEW

File: fluxette/js/components/CartContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import { connect } from '../flux';
+import { buyProducts } from '../flux/async';
+import Cart from '../../../common/components/Cart.jsx';
+
+@connect(state => state.cart, 'cart')
+export default class extends React.Component {
+    render() {
+        let { cart } = this.state;
+        cart = Object.keys(cart).map(key => cart[key]);
+        let total = cart.reduce((sum, product) => sum + product.price * product.quantity, 0).toFixed(2);
```
Comment (voronianski) [inline fluxette/js/components/CartContainer.jsx:11]: @edge why calculations in component?

File: fluxette/js/flux/stores.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { Store } from 'fluxette';
+import { CART, API } from './types';
+import { splicefilter } from '../util';
+
+let products = Store([], {
+    [API.PRODUCTS.DONE]: (products, action) => action.products,
+    [CART.ADD]: (products, action) => {
+        let { id } = action.product;
+        let [left, product, right] = splicefilter(products, p => p.id === id);
+        return [...left, { ...product, inventory: product.inventory - 1 }, ...right];
```
Comment (voronianski) [inline fluxette/js/flux/stores.js:10]: @edge after the bought of all items there should be `inventory: 0` (or not present in output) in console product but now it stays the same -

**Comment by voronianski** [comment]
@edge please consider fixing the issue with inventory in order to make example in sync with all others.

File: fluxette/js/util.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+// Order-preserving single filter
+export let splicefilter = (iter, pred) => {
```
Comment (voronianski) [inline fluxette/js/util.js:2]: @edge a bit confused why is that needed?

Comment (alkhe) [inline fluxette/js/util.js:2]: I could use two lines of Array#filter, but that's inefficient, so I wrote a small algorithm instead.
